### ISSUE TITLE
Link deck card to a contact 

### ISSUE THREAD

**Issue by guidtz** [nextcloud/contacts#2544]
Hello deck users and developpers,
 a customer asked us for a feature in deck cards.

**Describe the solution you'd like**
We would like to add in a card a link to a contact card in contact application.

Do you plan this ? 

Regards

**Comment by stefan-niedermann** [comment]
Hello,

this has to be implemented at the contacts app since the deck app already supports linking cards to projects. I moved the issue therefore :)

**Comment by guidtz** [comment]
You think that I will be a features to implement in contact, to link a contact with a deck card ?

**Comment by crispyduck00** [comment]
Hi. Any plan to implement this?

It would be so great to be able to link contacts to deck cards.

**Comment by fossil-free** [comment]
If I understand stefan-niedermann correctly, contacts would need to support nextcloud projects.
Afterwards it would be easy to add contacts to cards in the deck app (which I would very much appreciate).

Supporting nextcloud projects for contacts would be useful in any case, e.g. attaching contacts to specific files or such.

@miaulalala is there any plans on developing the project feature?

**Comment by ChristophWurst** [comment]
There are no plans for the near future.

**Comment by CoYoNq** [comment]
This feature have a lot of potential. Creating a task in a deck, associated to a contact, can improve a lot the way decks can be used/organized.
Sorry i can't collaborate with code (my skills in php are very limited), but as a system administrator, i know (they ask to), many of my users gonna love to have this feature.

**Comment by sunjam** [comment]
Projects functionality deprecated in nc25.

- https://github.com/nextcloud/server/issues/28320

**Comment by jcharaoui** [comment]
If Nextcloud is killing Projects, would it still be feasible for Deck to bring its own method of linking elements like Files, Contacts, Events, etc. into decks and cards? Deck could become the [Projects manager](https://github.com/nextcloud/server/issues/15841) that never was...

**Comment by wargreen** [comment]
Hi, since the link by Project not possible anymore, is it a way to link or embed a contact in a deck ?
It would be very useful for work and collaborate with peoples, inside or outside of the project !

Thanks

**Comment by mzramna** [comment]
it would be very usefull to me and my company, it could be implemented?

**Comment by ChristophWurst** [comment]
Yes, of course, @mzramna please drop us a message at https://nextcloud.com/get-a-quote/
